src/test/java/com/epam/tc/hw2/ex1/ExerciseOneTest.java

```diff
@@ -0,0 +1,121 @@
+package com.epam.tc.hw2.ex1;
+
+import com.epam.tc.hw2.MainTest;
+import java.util.Arrays;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class ExerciseOneTest extends MainTest {
```

Comment:
We should name the test classes and test methods according to what we're checking in them, please, try to think of better namings